config/routes.rb

```diff
@@ -52,6 +52,7 @@
 
   resource :admin, only: :show
   namespace :admin do
+    resources :druid_searches, only: :index
```

Comment:
Please remove the old search route too.

Reply:
This is the new route for the new item in the dropdown.

Reply:
Oh wait, do you mean line 18?

Reply:
I've removed the searches route.

Reply:
Looks good.